Extension/package.json

```diff
@@ -38,7 +38,10 @@
         "Snippets"
     ],
     "enabledApiProposals": [
-        "terminalDataWriteEvent"
+        "terminalDataWriteEvent",
+        "chatVariableResolver",
```

Comment:
I see these errors: 
>This proposal cannot be used because for this extension the product defines a fixed set of API proposals. You can test your extension but before publishing you MUST reach out to the VS Code team.

Is that an issue or what?

Reply:
I've reached out to the vscode team (Isidor) to get that approval. Planning to wait for that approval before pushing.

Reply:
Updated to just require the one necessary proposal, and have received the approval for it. Just waiting to be put on the allow list.